app/pages/complete-order/template.njk

```diff
@@ -15,6 +15,14 @@
       <div class="nhsuk-grid-column-two-thirds">
         <h1 data-test-id="complete-order-page-title" class="nhsuk-u-margin-top-7 nhsuk-u-margin-bottom-3">{{ title }}</h1>
         <h2 data-test-id="complete-order-page-description" class="nhsuk-u-font-size-24 nhsuk-u-margin-bottom-6">{{ description }}</h2>
+
+        {% if warningAdvice %}
+          <div data-test-id="complete-order-page-warning-advice" class="nhsuk-inset-text nhsuk-u-padding-top-2 nhsuk-u-padding-right-4 nhsuk-u-padding-bottom-2 nhsuk-u-padding-left-4 nhsuk-u-margin-top-5 nhsuk-u-margin-bottom-6">
+            <span class="nhsuk-u-visually-hidden">Information: </span>
+            <p class="nhsuk-u-font-size-14">{{ warningAdvice }}</p>
+          </div>
```

Comment:
any reason for not using the `insetText` component instead?

Reply:
Didn't know about it 😳 Will have a look, good spot 👍